src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.softserve.actent.service.impl;
+
+import com.softserve.actent.model.entity.Review;
+import com.softserve.actent.repository.ReviewRepository;
+import com.softserve.actent.service.ReviewService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewServiceImpl implements ReviewService {
+
+    @Autowired
+    ModelMapper modelMapper;
+
+    @Autowired
+    ReviewRepository reviewRepository;
+
+    @Override
+    public Review addReview(Review review) {
+
+        return reviewRepository.save(review);
+    }
+
+    @Override
+    public Review getReviewById(Long reviewId) {
+
+        if (reviewRepository.findById(reviewId).isPresent()) {
+            return reviewRepository.findById(reviewId).get();
+        } else {
+            return null;
+        }
+    }
+
+    @Override
+    public Review updateReviewById(Review review, Long reviewId) {
+
+        if (reviewRepository.findById(reviewId).isPresent()) {
+            review.setId(reviewId);
+            return reviewRepository.save(review);
+        } else {
+            return null;
```

Comment:
all those "return null" are a bit confusing, how to know from the client side that something went wrong?

read about ControllerAdvice

Reply:
We still wait for exceptions mechanism implementation, I think, we'll rework this correctly today.